src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -207,7 +207,7 @@ private async Task LoadCommitsTime(Docset docset)
             {
                 var (_, content, _) = await RestoreMap.GetRestoredFileContent(docset, docset.Config.Contribution.GitCommitsTime);
 
-                foreach (var commit in JsonUtility.Deserialize<GitCommitsTime>(content).Commits)
+                foreach (var commit in JsonUtility.DeserializeData<GitCommitsTime>(content).Commits)
```

Comment:
`DeserializeData` and `Deserialize` is not clear than previous `Deserialize` and `DeserializeWithSchemaValidation` from the method name.

Reply:
Both are fine, as long as they are consistent, right now JToken parse with schema validation is also called `(Error, JToken) Deseriaize(string)`. I tend to want to make  `DeserializeWithSchemaValidation` the default behavior for docfx to detect more warnings, so called it `Deserialize`

Reply:
https://github.com/dotnet/docfx/blob/7e9e7db60909ad1ea80fe22d2a6b7a4e90029c0d/src/docfx/lib/YamlUtility.cs#L82
https://github.com/dotnet/docfx/blob/7e9e7db60909ad1ea80fe22d2a6b7a4e90029c0d/src/docfx/lib/YamlUtility.cs#L87
These two were missing during the last renaming

Reply:
From the usage of these two methods, seems `DeserializeData` is used more than `Deserialize`, apart from the tests.